projects/elife.yaml

```diff
@@ -1975,12 +1976,10 @@ elife-xpub:
             device: /dev/sdh
             type: gp2
     aws-alt:
-        demo:
-            s3:
-                "{instance}-elife-xpub":
         staging:
```

Comment:
Is `prod` getting added in another PR?

Reply:
This is the deletion of `demo` which is configured but not used; both `staging` and `prod` have been configured to use `encryption`, look at the next additions in this file.